src/widget.ts

```diff
@@ -77,6 +77,8 @@ export const widget = ((): WidgetHelpers | null => {
       logger.info("Widget API is available");
       const api = new WidgetApi(widgetId, parentOrigin);
       api.requestCapability(MatrixCapabilities.AlwaysOnScreen);
+      api.requestCapabilityToSendEvent(EventType.CallEncryptionKeysPrefix);
+      api.requestCapabilityToReceiveEvent(EventType.CallEncryptionKeysPrefix);
```

Comment:
Is there a matching PR yet for matrix-react-sdk to auto-approve these capabilities?

Reply:
Yep, https://github.com/matrix-org/matrix-react-sdk/pull/11783 (and another previously from Simon adding them)